freedomofpress/securedrop
# Update translations for 1.0.0

## Status

Ready for review

## Description of Changes

Updated translations for 1.0.0.

## Testing

- Check out this branch.
- Run the page layout tests: `make test TESTFILES=tests/pageslayout`
- Check some of the screenshots under `securedrop/tests/pageslayout/screenshots/`
- Run the development server and check the strings changed for 1.0.0 by visiting with a couple of languages and verifying that the strings are in fact translated. The source start page has several, e.g. "First submission" and "Return visit".

## Deployment

No deployment concerns.

## Checklist

### If you made non-trivial code changes:
